### PR TITLE
Alpakka 2.0

### PR DESCRIPTION
**Pull request by ennru**
References https://github.com/akka/alpakka/issues/2276

### CODE REVIEW

File: blog/_posts/2020-04-30-alpakka-2.0.0-released.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+The Alpakka contributors are happy to announce **Alpakka 2.0**!
+
+## What is Alpakka?
+
+The [Alpakka project](https://doc.akka.io/docs/alpakka/current/) is an open source initiative to implement stream-aware and reactive integration pipelines for Java and Scala. It is built on top of [Akka Streams](https://doc.akka.io/docs/akka/current/stream/index.html), and has been designed from the ground up to understand streaming natively and provide a DSL for reactive and stream-oriented programming, with built-in support for backpressure. Akka Streams is a [Reactive Streams](https://www.reactive-streams.org/) and JDK 9+ [java.util.concurrent.Flow](https://docs.oracle.com/javase/10/docs/api/java/util/concurrent/Flow.html)-compliant implementation and therefore [fully interoperable](https://doc.akka.io/docs/akka/current/general/stream/stream-design.html#interoperation-with-other-reactive-streams-implementations) with other implementations.
+
+## About 2.0
+
+Things have taken much longer than we initially hoped and we hope the earlier milestone releases made it possible for the impatient users of Alpakka to try out the current versions and fixes.
```
Comment (patriknw) [inline blog/_posts/2020-04-30-alpakka-2.0.0-released.md:20]: do we need to excuse ourselves?

File: blog/_posts/2020-04-30-alpakka-2.0.0-released.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+The major version tick for Alpakka is due to major upgrades and changes in the dependencies used by the various connectors. Some Alpakka APIs changed as well, but we try to keep those changes minimal.
+
+**Alpakka 2.0 updates dependencies for most connectors.**
+
+**Alpakka 2.0 requires Akka 2.5.31 or Akka 2.6.4.** We recommend to upgrade your projects to the latest versions of Akka. Akka stays [binary-compatible](https://doc.akka.io/docs/akka/current/common/binary-compatibility-rules.html#binary-compatibility-rules) between patch releases (and for the most part between 2.5.x and 2.6.x).
```
Comment (patriknw) [inline blog/_posts/2020-04-30-alpakka-2.0.0-released.md:26]: 2.6.4 or later.

File: blog/_posts/2020-04-30-alpakka-2.0.0-released.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+
+## Major new functionality
+
+- Pravega connector [#2149](https://github.com/akka/alpakka/issues/2149) by [@cheleb](https://github.com/cheleb)
+   
+    [Pravega](http://www.pravega.io/) provides a new storage abstraction - a stream - for continuous and unbounded data. A Pravega stream is an elastic set of durable and append-only segments, each segment being an unbounded sequence of bytes. Streams provide exactly-once semantics, and atomicity for groups of events using transactions.
+
+- Cassandra: rewrite on top of Java driver 4.5.0 [#2182](https://github.com/akka/alpakka/issues/2182) by [@ennru](https://github.com/ennru)
+    **Alpakka Cassandra is completely rewritten and now based on the Cassandra Java driver version 4.6. It now becomes the base for [Akka Persistence Cassandra](https://doc.akka.io/docs/akka-persistence-cassandra/1.0/).**
```
Comment (patriknw) [inline blog/_posts/2020-04-30-alpakka-2.0.0-released.md:44]: in sentence above it says 4.5.0

File: blog/_posts/2020-04-30-alpakka-2.0.0-released.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    - Added flow for creating file ZIP archive [#1890](https://github.com/akka/alpakka/pull/1890) by [@michalbogacz](https://github.com/michalbogacz)
+
+We've adapted the Alpakka APIs even more to enable the use of the new actor APIs where possible.
+
+Read the full [Release notes](https://doc.akka.io/docs/alpakka/2.0/release-notes/2.0.x.html) for all details.
+
+
+## The way forward
+
+The Alpakka community is encouraged to continue to suggesting improvements and new functionality to the existing connectors, or even suggest integrations with other technologies. During Alpakka 2.0 we will keep the APIs binary-compatible.
```
Comment (patriknw) [inline blog/_posts/2020-04-30-alpakka-2.0.0-released.md:63]: "During Alpakka 2.0 patch releases" or 2.0.x

File: blog/_posts/2020-04-30-alpakka-2.0.0-released.md (hunk 1 above)
Comment (seglo) [inline blog/_posts/2020-04-30-alpakka-2.0.0-released.md:20]:
```suggestion
Things have taken much longer than we initially planned, but we hope the earlier milestone releases made it possible for our early adopters of Alpakka to try out the current versions and fixes.
```

File: blog/_posts/2020-04-30-alpakka-2.0.0-released.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    - Added flow for creating file ZIP archive [#1890](https://github.com/akka/alpakka/pull/1890) by [@michalbogacz](https://github.com/michalbogacz)
+
+We've adapted the Alpakka APIs even more to enable the use of the new actor APIs where possible.
+
+Read the full [Release notes](https://doc.akka.io/docs/alpakka/2.0/release-notes/2.0.x.html) for all details.
+
+
+## The way forward
+
+The Alpakka community is encouraged to continue to suggesting improvements and new functionality to the existing connectors, or even suggest integrations with other technologies. Alpakka 2.0 patch releases we will keep the APIs binary-compatible.
```
Comment (seglo) [inline blog/_posts/2020-04-30-alpakka-2.0.0-released.md:63]:
```suggestion
The Alpakka community is encouraged to continue to suggest improvements and new functionality to the existing connectors, or even suggest integrations with other technologies. Alpakka 2.0 patch releases we will keep the APIs binary-compatible.
```

**Pull request merged**